.ci/pitest.sh

```diff
@@ -9,7 +9,7 @@ function checkPitestReport() {
   SEARCH_REGEXP="(span  class='survived'|class='uncovered'><pre>)"
   grep -irE "$SEARCH_REGEXP" target/pit-reports \
      | sed -E 's/.*\/([A-Za-z]+.java.html)/\1/' | sort > target/actual.txt
-  printf "%s\n" "${ignored[@]}" | sed '/^$/d' > target/ignored.txt
+  printf "%s\n" "${ignored[@]}" | sed '/^$/d' | sort > target/ignored.txt
```

Comment:
please explain this change

Reply:
When I run the mutation tests locally on my laptop, I get random failures. I noticed then that in this script you are sorting "target/actual.txt" but not "target/ignored.txt". After I applied sorting on "target/ignored.txt", my problems were gone. To me it felt logical that if you sort one part of the comparison, the other part should also be sorted, it's more symmetrical this way.

Reply:
@ngeor ,
interesting point ..... "target/ignored.txt" suppose to be sorted, to keep diff on ignored under easy control.
Please try to revert this change(try one more time) and share details in separate issue.

Reply:
Sure thing I will give it a try.